src/train.jl

```diff
@@ -355,3 +355,51 @@ function sciml_train(loss, opt::BBO = BBO(), data = DEFAULT_DATA;lower_bounds, u
                                         NaN,
                                         bboptre.elapsed_time)
 end
+
+function sciml_train(loss, opt::TikTak, data = DEFAULT_DATA;lower_bounds, upper_bounds, local_method,
+                      maxiters = get_maxiters(data), kwargs...)

```

Comment:
any way to do callback support?

Reply:
No, there does not seem to be any callback support.